src/url.js

```diff
@@ -547,6 +549,17 @@ export function resolveRelativeUrl(relativeUrlString, baseUrl) {
   return resolveRelativeUrlFallback_(relativeUrlString, baseUrl);
 }
 
+/**
+ * Returns absolute URL resolved based on the relative URL and the element.
+ * @param {string} relativeUrlString
+ * @param {!AmpElement} element
```

Comment:
Any attached element is fine here.
```suggestion
 * @param {!Element} element
```